src/server/wesnothd/server.cpp

```diff
@@ -1807,7 +1807,33 @@ void server::handle_player_in_game(socket_ptr socket, simple_wml::document& data
 	} else if(data.child("stop_updates")) {
 		g.send_data(data, socket);
 		return;
-		// Data to ignore.
+	} else if(data.child("game_history_request")) {
```

Comment:
You could assign the child in one go same as a couple lines up and then use that instead of calling `child()` again.

Reply:
Updated.